iothub/service/src/FeedbackBatch.cs

```diff
@@ -11,10 +11,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// A timestamp that indicates when the feedback message was received by the IoT hub.
+        /// </summary>
         public DateTime EnqueuedTime { get; set; }
 
+        /// <summary>
+        /// A collection of feedback records across multiple devices and messages for an IoT hub.
+        /// </summary>
         public IEnumerable<FeedbackRecord> Records { get; set; }
 
+        /// <summary>
+        /// The Iot Hub name is the user Id.
```

Comment:
For real?

Reply:
I was also surprised

Reply:
This was in the msdn docs. Unless that is wrong.

Reply:
Anyone has any idea if this is incorrect?

Reply:
In references this is what I see
` UserId = Encoding.UTF8.GetString(amqpMessage.Properties.UserId.Array, amqpMessage.Properties.UserId.Offset, amqpMessage.Properties.UserId.Count)`

Reply:
We can run a sample and confirm

Reply:
Would be great. Let us sync offline and resolve this.

Reply:
ran the sample, this value is really a hostname of IoT Hub. Pretty bad naming :(

Reply:
Thanks @bikamani  for testing and confirming that it is the IoT hub hostname in the userId field. I will updated this as decided - Name of the IoT hub and in remarks call it out.